superset-frontend/src/SqlLab/components/TableElement/index.tsx

```diff
@@ -107,7 +108,7 @@ const TableElement = ({ table, ...props }: TableElementProps) => {
   const {
     currentData: tableMetadata,
     isSuccess: isMetadataSuccess,
-    isLoading: isMetadataLoading,
+    isFetching: isMetadataLoading,
```

Comment:
haven't dug deep, but I'd assume switching the name here we'd fetch some other reference to it somewhere else?

Reply:
We are destructuring `isFetching` (which is already part of the return of `useTableMetadataQuery`) into `isMetadataLoading`.
(`isLoading` is only true on the first load while `isFetching` is also true on subsequent fetches.)

I renamed the local variable `isMetadataLoading` to `isMetadataFetching` to be in sync with the naming convention.